modules/installation-vsphere-machines.adoc

```diff
@@ -118,7 +118,7 @@ endif::openshift-origin[]
 .. Right-click the name of your datacenter.
 .. Click *New Folder* -> *New VM and Template Folder*.
 .. In the window that is displayed, enter the folder name. The folder name must
-match the cluster name that you specified in the `install-config.yaml` file.
+match the folder name that you specified in the `install-config.yaml` file or, if you did not specify a folder name, the infrastructure ID.
```

Comment:
Should we specify how to get infrastructure ID? 

We have instructions in AWS UPI: https://bz1849434--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-extracting-infraid_installing-aws-user-infra